### ISSUE TITLE
Merlin: Oscillating radial collimator: set soft limits on underlying motor

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#3607]
As an instrument scientists on merlin I don't want a motor to destroy itself if it is told to go to a location it can't reach.

Helen believes that the labview driver may have put soft limits on the underlying motor to prevent it from driving to locations it can't reach and destroying itself.

Note: physical motor is currently in bits in electronics lab

**Comment by GDH-ISIS** [comment]
I do not believe the LabVIEW driver used soft limits in the Galil to stop the underlying motor driving to locations it can't reach. I think the LabVIEW front panel control will have had limits applied to it so that the set point can only be entered within certain bounds.

**Comment by kjwoodsISIS** [comment]
Consider the points in #3611.
